### ISSUE TITLE
Facilities UX Sync agenda

### ISSUE THREAD

**Issue by kmariepat-cityfriends** [department-of-veterans-affairs/va.gov-cms#14087]
<details><summary>Meeting Purpose and Structure</summary>

This is a weekly meeting where we focus on the following **actions**:
- Presenting UX work for feedback / signoff
- Roadmapping UX work for prioritized initiatives
- Refining cards for upcoming UX work

**Agenda:** 
- The agenda is updated on a weekly basis by Dave Pickett
- Draft agenda will be sent out ~24 hours in advance via Slack
- If you have an item you would like to add, you can always add a comment to the ticket async
- Agenda items should have clear action-oriented language. Why are we looking at this item? What is the desired feedback or next step?

**Attendees:**
- Core attendees are Facilities UX team members and Facilities PM / DM / PO
- All other attendees are optional and may choose to join or skip based on agenda

_Note: 10/20/2023 – Updated meeting title and description_

</details>

# Agenda items for 12/21

## Main event
- [x] Overflow discussions from Q1 meeting
- [x] Touch base on #15631 before sending to CAIA / Midpoint
- [ ] This meeting scheduling in 2024

## Stretch or async
- [ ] (If ready) review proposals from #16415 
- [ ] FUTURE MEETING Discuss #16337

**Comment by jilladams** [comment]
Thurs 7/20: 
https://github.com/department-of-veterans-affairs/va.gov-team/issues/62212
https://github.com/department-of-veterans-affairs/va.gov-team/issues/62175

**Comment by jilladams** [comment]
### Fac Locator
I've moved content of this ticket body for FacLocator: into https://github.com/department-of-veterans-affairs/va.gov-cms/issues/11542

Facility Locator:

1. Get sign off on Designs internally (week of 6/19)
2. Kick off platform collab cycle with design intent meeting (week of 6/26)
3. Incorporate feedback from platform 
4. Pause on usability work until post VBA Editor Research


### VBA
 into https://github.com/department-of-veterans-affairs/va.gov-cms/issues/13029

VBA:

- [x] Final content from Sitewide Content
- [x] Designs: update with final content & sign off internally
- [ ] Review front end designs with VBA 
- [ ] Review Conversation guide & Usability testing Tasks re: final content / designs
- [ ] Review open questions from CMS collab touchbase on content specs 

Sprint 87
- [x] Review Service Taxonomy + other aspects of VBA design with Content 

Sprint 88
- [ ] Alexis to compile recommendations for UX Research Artifacts based on feedback from sitewide content 
- [x] Jordan to incorporate sitewide content feedback into designs 
- [ ] VBA Design sign off internally 
- [ ] Cindy to provide input design (+ prototype) upon return
- [ ] Cindy to review Alexis recommendations and finalize UX Research Artifacts

Stretch Sprint 88
- [ ] Have Midpoint Review with Platform 
- [ ] Start to build Prototype 

Sprint 89
- [ ] PO signoff on Research Plan + Conversation Guide 
- [ ] Research Ops Review w/ Shane or other team member 
- [ ] Kickoff meeting with Perigean 
- [ ] Begin Recruiting (allow 5 business days)
- [ ] Piloting 
- [ ] Send signup for observing to VBA ROs present this option in VBA RO meeting on 7/19

 Sprint 90
- [ ] Conduct Usability 
 
Sprint 91 
- [ ] Synthesize research + presentation created for VBA RO audience

**Comment by jilladams** [comment]
Keeping the ticket open for now, to keep track of what we need to next refine, until I'm more organized on that front.

**Comment by cindymerrill** [comment]
@jilladams This should NOT have been checked off for VBA because that's what's blocked:  
_Designs: update with final content & sign off internally_

**Comment by jilladams** [comment]
Thurs 7/20: 
https://github.com/department-of-veterans-affairs/va.gov-team/issues/62212
https://github.com/department-of-veterans-affairs/va.gov-team/issues/62175

* VBA content spec work -maybe alexis? [Content Spec Drupal work for VBA Regional Office MVP · Issue #13096 -0- Alexis may have capacity in this sprint
* Opp: synthesize previous editorial research / make recommendations
    * Jo did this to some extent as her first project, re: editors of CMS products broadly (meaning: Facilities, as Fac products are the only ones to have had Editor research conducted. e.g. COVID facility status, Blake has done some)
    * Jo's meta-analysis https://github.com/department-of-veterans-affairs/va.gov-cms/issues/13019
    * Alexis can familiarize herself as a starting point
* Who has capacity? What do we need to do to utilize it?

**Comment by mmiddaugh** [comment]
Regarding CC Design intent for the Facility Locator "Consider: is there any reason users may want to sort their search results on some criteria other than closest distance?"


We do have Veteran feedback on this topic, informed by [Facility Locator Discovery Research in November 2019](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/facilities/facility-locator/research/discovery-sprints/user-research/user-research-findings.md#summary): 
> Summary Finding 1. Veterans search by a specific service and by distance when choosing a location to visit. The Facility Locator proved most helpful when Veterans understood what services they could receive from locations nearest to them.

**Comment by mmiddaugh** [comment]
The following items look actionable before Midpoint review: 

- Replace modal with drawer for use with multiple filters
- Review [icons library](https://design.va.gov/foundation/icons) for suitable option to use for provider core training complete, 
- Investigate [US system segmented button group](https://designsystem.digital.gov/components/button-group/) as possible replacement for tabs and check on tech implementation
- Explore other placements for the button that opens the filter modal/dropdown

I added the accessibility feedback about map markers to a comment on [Improve usability of mobile map #12919](https://app.zenhub.com/workspaces/sitewide-facilities-639f5253e4b702a32376339e/issues/gh/department-of-veterans-affairs/va.gov-cms/12919#issuecomment-1650579191)

**Comment by jilladams** [comment]
Bless. Swapped narrative out of body here in favor of link to #14533 that contains this consolidated list.

**Comment by jilladams** [comment]
Thurs 7/27: 
- [x] Midpoint review artifact: signed off by Michelle, Cindy, Jordan
    - https://www.sketch.com/s/891d33ae-152d-471c-ab5d-9aedf89cf6ff
    - https://www.sketch.com/s/891d33ae-152d-471c-ab5d-9aedf89cf6ff/p/9BAF00BD-1CA9-4FCF-A14B-E2B3D31A18E6/canvas
- [x] Midpoint review ticket: https://github.com/department-of-veterans-affairs/va.gov-team/issues/46029
- VBA content tickets
- ~~VBA Research blockers~~
- [x] Facility Locator next steps / what's important

## VBA Regional Office Modernization MVP ([#10514](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/10514))
Who has capacity? What do we need to do to utilize it?

**Design**: 
Q. Other than clickable prototype, what's left for design? 
A. Editor interface, TBD decisions.

Q. What's smart to do in this vein, based on what we've learned on PW? 
A. Moving in lockstep with Drupal eng is nice, to confirm that Design isn't designing something that's not possible in Drupal, and that Drupal isn't building a bad UX. 

TODO: Ticket the editor experience Design, make a Design ticket to correspond to each of these CMS build tickets
- [ ]  https://github.com/department-of-veterans-affairs/va.gov-cms/issues/12597
- [ ]  https://github.com/department-of-veterans-affairs/va.gov-cms/issues/12626
- [ ]  https://github.com/department-of-veterans-affairs/va.gov-cms/issues/12618
- [ ]  https://github.com/department-of-veterans-affairs/va.gov-cms/issues/12596
- [ ]  https://github.com/department-of-veterans-affairs/va.gov-cms/issues/12616

**Content**: 
- #13799 
- #13811 
- #14529 
    - #13800 
- #11267 
  - Team [#58511](https://github.com/department-of-veterans-affairs/va.gov-team/issues/58511) (Services CAIA ticket) - close loop re: Content in open accordions after midpoint
    * Service accordions: Midpoint review "Home Loans accordion content - this is just one of three service accordions, we plan to have openable in the usability prototype. We would like to get sign off from Platform at Midpoint Review on the overall structure of the accordion interior and then create the https://github.com/department-of-veterans-affairs/va.gov-cms/issues/13960. Could we plan on having y'all look at the Home Loan & other 2 accordions content again once we're in the prototype stage? What are the tickets we will pull into sprint 90 to unblock"


**Research**: 
- Identify all Research blockers 
  - Understand the state of the Service Accordions
      - For research: testing content in the accordions is one of our Research goals. If we keep that, we need 1) A treatment of an open accordion, 2) with content (not lorem), tho content doesn't have to be final.
      - Service accordion copy is drafted (DaveP)
      - Midpoint review artifact: is confirming the structure of the design as Lorem, to reduce distractions
      - After Midpoint signoff on structure: we will get CAIA review of the draft content
      - When CAIA signs off: We will put that content into the prototype, and Research is unblocked.
  - [x] make sure any needed new tickets are refined for next sprint
  - Update flow in epic body (#10514)
* Veteran usability: #10197 now contains full set of research tickets (based on Income Limits tickets)
* Editor: 
    * https://github.com/department-of-veterans-affairs/va.gov-cms/issues/14528
    * Figure out what we are doing about #10185  

---

## Facility Locator  ([#11542](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/11542))
* #14533 
* Prep for veteran usability testing (Tickets are stubbed out / blank, and need to be populated with runbook contents or replaced with runbook versions)
---
## VAMC Hardening ([#12920](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/12920))
Harden VAMC Facility Map 
Harden Required Health Services 

**Product**
Need to: 
* Confirm: are Map & Required Health Services linked, or 2 separate efforts? 
* Identify tickets that exist
* Clean out into an MVP epic for this effort 

---

## Other priorities

**Comment by jilladams** [comment]
Drafted agenda for 8/3, based on open items from today, in ticket body.

**Comment by cindymerrill** [comment]
@davidmpickett @xiongjaneg @jilladams @mmiddaugh In preparation for tomorrow’s Facilities UX Refinement, I’ve updated the VBA [conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/facilities/regional-offices/research/2023-06-veteran-facing/conversation-guide.md) per the prototype and [research plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/facilities/regional-offices/research/2023-06-veteran-facing/research-plan.md) per today’s Option 3 timeline. Here are the details left to finalize for these two research deliverables before we can launch recruiting:

- [ ] Any changes to dates for kickoff meeting and research sessions
- [ ] Any changes to Alexis’ availability for the new research dates of Aug 29-31
- [ ] Any changes to the prototype, including in response to my 2 questions in [this ticket](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/14524#issuecomment-1672131278)
- [ ] Any feedback from anyone on this thread, including Michelle's approval
- [ ] Updated link to the prototype (when @thejordanwood  has one that doesn’t show hotspots)
- [ ] Link to card in research repo (need to create this; doesn't take long)
- [ ] Link to recruitment intake ticket (need to create this; doesn't take long)

**Comment by davidmpickett** [comment]
Just updated the main comment to reflect current state of affairs and expected discussion points for 8/17

**Comment by xiongjaneg** [comment]
@aklausmeier Noted, thank you. Added to AC.

**Comment by jilladams** [comment]
@davidmpickett @ALogsdon3  (cc @xiongjaneg ) Could we tomorrow also look at upcoming sprint and any queued tickets that Alexis might be comfortable picking up while Jordan is OOO next week?

**Comment by cindymerrill** [comment]
I'm going to remove myself as an Assignee on this ticket. Feel free to add me back on if I do work with Facilities in the future :)

**Comment by davidmpickett** [comment]
> Maybe someone beat me to it? I did just receive any notification about the last comment on this ticket.

Unsubscribing from notifications is technically a separate action from un-assignment. Anyone can assign/unassign, but only you can unsubscribe from notifications.

**Comment by cindymerrill** [comment]
@davidmpickett I just found the "unsubscribe" button on this ticket, and I clicked it! So now I'm all set :)

**Comment by davidmpickett** [comment]
Closing this in favor of a fresh ticket for 2024 #16586
